docs/user/README.md

```diff
@@ -18,6 +18,14 @@ By default, the cluster will be given the name `kind-1`. "1," here, is the
 default context name. 
 Use the `--name` flag to assign the cluster a different context name.
 
+**Note**: If you are running `kind` of MacOS or Windows then it is recommended
+you have at least 4GB of RAM and disk space dedicated to the virtual machine 
+(vm) running the Docker engine otherwise the Kubernetes cluster might fail to 
+start up.
+If you come across erros where `kubectl` cannot communicate with the cluster
+and you have increased the memory and size of the VM, then try cleaning up
+space by performing a `docker system prune`.
```

Comment:
i think this can be changed to:
> If you've experienced errors and had to increase the memory and disk space of the VM, don't forget to "prune" your container runtime - e.g. `docker system prune`.

Reply:
I fixed the wording somewhat. Does one "prune a container runtime"? 
To prune a container runtime makes me understand that there is something to change from the way a container is run.